---a/net/minecraft/block/BlockAir.java
+++b/net/minecraft/block/BlockAir.java
@@ -6,34 +6,29 @@
 import net.minecraft.util.BlockPos;
 import net.minecraft.world.World;
 
-public class BlockAir extends Block
-{
+public class BlockAir extends Block {
+
     private static final String __OBFID = "CL_00000190";
 
-    protected BlockAir()
-    {
+    protected BlockAir() {
         super(Material.air);
     }
 
-    public int getRenderType()
-    {
+    public int getRenderType() {
         return -1;
     }
 
-    public AxisAlignedBB getCollisionBoundingBox(World worldIn, BlockPos pos, IBlockState state)
-    {
+    public AxisAlignedBB getCollisionBoundingBox(World worldIn, BlockPos pos, IBlockState state) {
         return null;
     }
 
-    public boolean isOpaqueCube()
-    {
+    public boolean isOpaqueCube() {
         return false;
     }
 
-    public boolean canCollideCheck(IBlockState state, boolean hitIfLiquid)
-    {
+    public boolean canCollideCheck(IBlockState state, boolean hitIfLiquid) {
         return false;
     }
 
-    public void dropBlockAsItemWithChance(World worldIn, BlockPos pos, IBlockState state, float chance, int fortune) {}
-}
+    public void dropBlockAsItemWithChance(World worldIn, BlockPos pos, IBlockState state, float chance, int fortune) {
+    }
+}
